observability/log-vs-metric.md

```diff
@@ -0,0 +1,73 @@
+# Logs vs Metrics
+
+## Overview
+
+### Metrics
```

Comment:
Is the plan to consolidate the same information available in this page: https://github.com/microsoft/code-with-engineering-playbook/blob/master/observability/pillars/metrics.md or is it OK to keep them separate?

Reply:
It is fine to keep them separate, provided they have similar/complimenting messaging, I think.